container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -3128,78 +3033,13 @@ variables:
     required: true
-- name: AUTOSCALER_API_SERVER_CACHE_TTL
-  options:
-    default: 600
-    description: "The TTL of credential cache for the Autoscaler custom metrics, in seconds"
-- name: AUTOSCALER_API_SERVER_DB_CONFIG_IDLE_TIMEOUT
-  options:
-    default: 100
-    description: Idle connection timeout for the Autoscaler API Server database, in
-      seconds.
-- name: AUTOSCALER_API_SERVER_DB_CONFIG_MAX_CONNECTIONS
-  options:
-    default: 10
-    description: The maximum number of connections to the Autoscaler API Server database.
-- name: AUTOSCALER_API_SERVER_DB_CONFIG_MIN_CONNECTIONS
-  options:
-    default: 0
-    description: The minimum number of connections to the Autoscaler API Server database.
-- name: AUTOSCALER_API_SERVER_INFO_BUILD
-  options:
-    default: beta
-    description: The build version of Autoscaler. It should be defined when it is
-      deployed to a product environment.
-- name: AUTOSCALER_API_SERVER_INFO_DESCRIPTION
-  options:
-    default: autoscaler
-    description: The description of Autoscaler. It should be defined when it is deployed
-      to a product environment.
-- name: AUTOSCALER_API_SERVER_INFO_NAME
-  options:
-    default: autoscalerapiserver
-    description: The name of the Autoscaler API Server. It should be defined when
-      it is deployed to a product environment.
-- name: AUTOSCALER_API_SERVER_INFO_SUPPORT_URL
-  options:
-    default: ''
-    description: The support url of autoscaler where the users could find support
-      information. It should be defined when it is deployed to a product environment.
-- name: AUTOSCALER_APPMETRICS_DB_CONNECTION_CONFIG_CONNECTION_MAX_LIFETIME
-  options:
-    default: 60s
-    description: The maximum age of a connection that can be reused for the Autoscaler
-      appmetrics database.
-- name: AUTOSCALER_APPMETRICS_DB_CONNECTION_CONFIG_MAX_IDLE_CONNECTIONS
-  options:
-    default: 10
-    description: The maximum number of connections to the Autoscaler appmetrics database.
-- name: AUTOSCALER_APPMETRICS_DB_CONNECTION_CONFIG_MAX_OPEN_CONNECTIONS
-  options:
-    default: 100
-    description: The minimum number of connections to the Autoscaler appmetrics database.
-- name: AUTOSCALER_ASAPI_CLIENT_CERT
-  options:
-    secret: true
-    ca: INTERNAL_CA_CERT
-    alternative_names:
-    - as_api_client
-    description: A PEM-encoded TLS certificate for clients to connect to the Autoscaler
-      API. This includes the Autoscaler ApiServer and the Service Broker.
-    required: true
-  type: certificate
-- name: AUTOSCALER_ASAPI_CLIENT_CERT_KEY
-  options:
-    secret: true
-    description: A PEM-encoded TLS key for clients to connect to the Autoscaler API.
-      This includes the Autoscaler ApiServer and the Service Broker.
-    required: true
 - name: AUTOSCALER_ASAPI_PUBLIC_SERVER_CERT
   options:
     secret: true
     ca: INTERNAL_CA_CERT
     alternative_names:
     - autoscaler.{{.DOMAIN}}
+    - localhost
```

Comment:
I'm a little surprised how many certs need a `localhost` alt name. Not really a problem, but I would like to understand why this was done.

Reply:
already answered in previous comment so resolving this one